SpaceDash/Extensions/String + Extension.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
Can you rename the file as just String.swift?

Reply:
This is actually bad practice and it will misguide other that this file "string.swift" will contain a class for String. Always use String + Extension (classname + Extension) so that it will be easy for others to know what the file is and what is implemented within.

Reply:
I have changed the filename in order to retain consistency.